engine/time/src/main/java/io/deephaven/time/calendar/BusinessCalendar.java

```diff
@@ -1357,14 +1361,16 @@ public LocalDate[] nonBusinessDates(final LocalDate start, final LocalDate end,
      * @throws InvalidDateException if the dates are not in the valid range
      * @throws DateTimeUtils.DateTimeParseException if the string cannot be parsed
      */
-    public LocalDate[] nonBusinessDates(final String start, final String end, final boolean startInclusive,
+    public String[] nonBusinessDates(final String start, final String end, final boolean startInclusive,
```

Comment:
Same comment as the one on `businessDates`

Reply:
I assumed that String in implies String out.  If someone is working in strings, do you expect that they would not want this result in strings as well?